src/glossary.rst

```diff
@@ -967,3 +967,117 @@ Glossary
       See also:
 
       * :ref:`Graph Branching`
+
+source workflow
+   A source workflow is a :term:`cylc workflow` that is not located in the
+   :term:`cylc run directory`.
+
+   In order to be :term:`run` source workflows must be
+   :term:`installed <install>` into the :term:`cylc run directory`.
+
+workflow
+cylc workflow
+   A Cylc workflow is a collection of things to run (:term:`tasks`) along with
+   the constraints that govern the order in which they run
+   (:term:`dependencies`) represented in Cylc format in a
+   :cylc:conf:`flow.cylc` file.
+
+   For example here is a Cylc workflow representing the brewing process:
```

Comment:
🤣 How very you....

Reply:
Perhaps we should jot some abstract examples down on cylc-admin:
Off the top of my head:
- Cooking
- Gardening 
- DIY
- Start of Driving lesson (chair, seatbelt, mirrors, hands at 10 and 2, clutch (if manual), choke(if old car), start, mirror, signal, maneuver, possible branch )
-

Reply:
> a disturbingly small proportion of the population are familiar with the brewing process

But a gratifying large proportion are familiar with the products thereof :beers: